src/postcapture.rs

```diff
@@ -10,6 +10,12 @@ use crate::{
     historymodel::{ModelNotifier, RowData},
 };
 
+pub fn do_postcapture_actions(history_model: &HistoryModel, conn: &SqliteConnection, pixbuf: &mut Pixbuf) {
```

Comment:
Nitpick: maybe we could call `run_postcapture_actions`? Not sure if I'm that happy with this name